content/05.use-cases-c.md

```diff
@@ -2,66 +2,58 @@
 
 ### Project continuity
 
-<!--*Contributors to this section: BPME, VF*  -->
+<!--*Contributors to this section: BPME, VF, PHPB  -->
 
-The development of research code continues, and so does the need to consider project continuity, especially in EEB where graduate students, research assistants, and postdoctoral fellows often hold relatively short-term positions [@doi:10.5334/jors.307].
-Often with these types of short-term research projects, once the contract expires, the research code upkeep tends to fall off as the researchers move on to new projects.
-Additionally, if the code is kept on only the researcher's personal devices, it becomes increasingly difficult to access the code for future uses.
-As projects or contracts end, there should be a handover period of this code in order for the next cohort of researchers to reuse what was already developed [@doi:10.5334/jors.307; @doi:10.1186/1751-0473-8-7].
+Projects in ecology and evolution often extend beyond the residence of graduate students, research assistants and post-doctoral fellows, who hold limited-term positions [@doi:10.5334/jors.307].
+Without clear plans on project continuitiy, the research code and data management upkeep tends to fall off as researchers move on to new projects or to other institutions.
+Additionally, code and data can be difficult to access when kept only on the personal devices of researchers.
 
-GitHub facilitates project continuity among research code by providing tools that make this handover period easier. 
-As we have already mentioned, using Git for code in Ecology and Evolution can allow for a "paper trail" of sorts to be created for the code, thus allowing for future users of the code access to the entire history of the project [@doi:10.1038/s41559-017-0160]. 
-Additionally, GitHub allows for repositories and organizations to have designated Code Owners [@url:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners]; these code owners can change through time allowing for the transition of code from one cohort of researchers to the next [@doi:10.1890/ES14-00402.1].
-There are also multiple means by which to archive code dependency information with GitHub.
-Dependencies can be described in README files, added to environment.yml and/or an environment.txt file, or stored in a repository using tools such as renv (<https://rstudio.github.io/renv/articles/renv.html>).
-This ensures that as packages continue to develop with new syntax, code will still run as it did when it was developed.
+On one hand, GitHub can facilitate project continuity among research code by making code handover to succeding researchers easier [see @doi:10.5334/jors.307; @doi:10.1186/1751-0473-8-7]. 
```

Comment:
I think that the "on one hand" and later the "on the other hand" can be deleted without altering the message here

Reply:
Agreed, updated text.